bot/src/plugins/hangman.cr

```diff
@@ -134,10 +139,10 @@ class Hangman
     end
   end
 
-  def start_game msg, list
+  def start_game msg, list, guess_max
```

Comment:
Nah, I prefer to not restrict unless needed for some reason or when providing external API.